src/processing/ChunksProcessing.ts

```diff
@@ -99,7 +99,8 @@ export const chunksProcessingTaskHandler: ChunksProcessingTaskHandler = async (
   const chunks = [...lowerChunks, ...upperChunks]
   return skipBlobCompression
     ? chunks.map(chunk => chunk.toStub())
-    : await Promise.all(chunks.map(chunk => chunk.toCompressedBlob()))
+    // : await Promise.all(chunks.map(chunk => chunk.toCompressedBlob()))
+    : await chunksToCompressedBlob(chunks)
```

Comment:
We'll use the chunks array compression from the sdk since it takes less place and the dApp handles timings of compression/decompression, the blob system can be removed from the world